packages/framework/assets/js/admin/validation/form/freeTransportAndPayment.js

```diff
@@ -18,4 +18,4 @@ export default function validationFreTransportAndPayment () {
     });
 }
 
-(new Register()).registerCallback(validationFreTransportAndPayment);
+(new Register()).registerCallback(validationFreTransportAndPayment, 'validationFreTransportAndPayment');
```

Comment:
here it looks like a typo (fre X free). I'm not sure if the class should be renamed in this PR, but the key can be correct even now, or?

Reply:
745c39a79812facb4e2922516390a1c9e137086a